engines/hpl1/credits.pl

```diff
@@ -0,0 +1,3 @@
+begin_section("Hpl1");

```

Comment:
we need fill entries here. @sev-  what you could suggest to put here?

Reply:
Emanuele Grisenti, perhaps @digitall (David Turner). My contribution is not significant. also, perhaps @aquadran (Paweł Kolodziejski)

Reply:
My contributions to HPL1 inclusion are mainly minor fixes for warnings, though I do have a copy of the game so looking forward to playing reliably (the original is twitchy about OpenGL drivers / cards resulting in glitches when I tried to play it on Win32).

Reply:
If you want to add me, feel free, but up to @grisenti who has done most of the heavy work here :)